dockers/docker-saiserver-brcm/Dockerfile

```diff
@@ -17,8 +17,7 @@ COPY ["deps/saiserver", "start.sh", "/usr/bin/"]
 COPY ["profile.ini", "portmap.ini", "/etc/sai/"]
 
 ## Clean up
-RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
-RUN rm -rf /deps
+RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y; rm -rf /deps
```

Comment:
i think this is okay to leave it here as a specific step to clean everything up.